unittest/tofdm.c

```diff
@@ -208,6 +208,10 @@ int main(int argc, char *argv[])
     ofdm = ofdm_create(ofdm_config_default);
     assert(ofdm != NULL);
     ofdm_config = ofdm_get_config_param();
+
+    /////////////// needs option added above
+    ofdm_set_phase_est_bandwidth_mode(ofdm, HIGH_PHASE_EST);
+    ///////////////
```

Comment:
This shouldn't be necessary - ofdm_create() should start in HIGH_PHASE_EST, as per the Octave code.  The tofdm system exercises both high and low modes as the state machine "switches gears" as the modem runs and achieves sync.

Reply:
Thanks Steve - looks good.

I wonder of it's possible to select high/low/auto phase est bandwidth selection through ofdm API calls and via command line options in ofdm_demod?  If the option is not auto, we would need to disable the state machine switching logic.  However just a thought - not something we really need right now.

The parallel/wide range freq offset estimation code at line 436 of ofdm_lib.m needs porting to C if you're interested, otherwise I'll get around to it this weekend.  I'm pretty happy with this - finally a wider range for freq offset acquisition on this modem :-)